cypress/integration/load_safe.spec.js

```diff
@@ -0,0 +1,65 @@
+        cy.visit("/")
+        cy.findByText('Accept selection').click()
+
+        cy.get('[data-track="load-safe: Open stepper"]').click()
+        cy.findByText('Add existing Safe').should('exist')
+        cy.wait(1000) //Have to wait because clicking the switch network fails sometimes if not
+
+        //Step 1
+        cy.get('[data-testid="select-network-step"]').find('button').click()
+        cy.get('[aria-labelledby="select-network"]').should('exist')
+        cy.findByText('Ethereum').click()
+        cy.get('nav').findByText('Ethereum').should('exist')
+        cy.get('[data-testid="select-network-step"]').findByText('Ethereum').should('exist')
+        cy.get('[data-testid="select-network-step"]').find('button').click()
+        cy.findByText('Rinkeby').click()
+        cy.get('[data-track="load-safe: Continue"]').click()
+
+        //Step2
+        cy.get('[data-testid="load-safe-name-field"]').should('have.attr', 'placeholder').should('contain','rinkeby-safe')
+        cy.get('[data-testid="load-safe-name-field"]').type('Test safe name').should('have.value', 'Test safe name')
+
+        cy.get('[data-testid="load-safe-address-field"]').type('RandomText')
+        cy.findByText(INVALID_INPUT_ERROR_MSG).should('exist')
+        cy.get('[data-testid="load-safe-address-field"]').clear().type(NON_CONTRACT_ADDRESS)
+        cy.findByText(INVALID_ADDRESS_ERROR_MSG).should('exist')
+        cy.get('[data-testid="load-safe-address-field"]').clear().type(SAFE_ENS_NAME).should('have.value', SAFE_ENS_NAME_TRANSLATED)
+        cy.get('[data-testid="qr-icon"]').click()
+        cy.get('[class="paper"]').find('button').contains('Upload an image').click()
+        cy.get('[type="file"]').attachFile('../utils/files/rinkeby_safe_QR.png')
+        cy.get('[data-testid="load-safe-address-field"]').should('have.value', SAFE_QR_CODE_ADDRESS)
+        cy.get('[data-testid="safeAddress-valid-address-adornment"]').should('exist')
+        cy.wait(3000) //have to wait or after clicking next what loads is the owners of the previous valids safe, not the one from the QR code
+        cy.get('[type="submit"]').click()
+
+        //Step3
+        cy.findByPlaceholderText(OWNER_ENS_DEFAULT_NAME).parents('div[data-testid="owner-row"]').findByText(OWNER_ADDRESS)
+        cy.findByPlaceholderText(OWNER_ENS_DEFAULT_NAME).type('Test Owner Name').should('have.value', 'Test Owner Name')
+        cy.get('[data-track="load-safe: Continue"]').click()
+
+        //Review Step
+        cy.findByText('Test safe name').should('exist')
+        cy.findByText('Test Owner Name').should('exist')
+        cy.get('[data-track="load-safe: Add"]').click()
+
+        //Safe Loaded
+        cy.get('[data-testid="sidebar"]').findByText('Test safe name')
+        cy.get('[data-testid="sidebar"]').find('nav').findByText('Settings').click()
+        cy.get('[data-testid="sidebar"]').find('nav').findByText('Owners').click()
+        cy.findByText('Test Owner Name')
+    })
```

Comment:
Looks like this scenario tests only the happy path.
It would be nice to also check possible input errors, Safes that don't exist (or on a diff network), adding and discarding owners, going back etc.
Doesn't need to be in the PR, just an idea for future.